docs/volunteering/tasks.md

```diff
@@ -0,0 +1,61 @@
+# Tasks
+
+We record tasks which need doing at https://github.com/srobo/tasks/issues.
+
+If you spot a task which you'd like to do or help with, please `@mention`
```

Comment:
is it worth linking to https://help.github.com/en/articles/basic-writing-and-formatting-syntax#mentioning-people-and-teams ? I'm not sure how common @mentions are outside of github and slack.

(And the 2 times below too)

Reply:
`@mention`s worked on IRC and I'm fairly sure they work on Twitter?
That's a useful link to add though: ce9b8b9